packages/printer-legacy/src/printer.js

```diff
@@ -193,12 +198,22 @@ class Printer {
     });
     const description = Printer.printDescription(type);
     const code = Printer.printCode(type, printTypeOptions);
+    const constDirectives = Printer.printConstDirectives(
```

Comment:
Nitpick. (second comment) why `constDirectives`  and `Printer.printConstDirectives` instead of `customXXXX`?